tox.ini

```diff
@@ -539,7 +550,7 @@ commands_pre =
   python -m pip install -e {toxinidir}/instrumentation/opentelemetry-instrumentation-asgi[test]
   python -m pip install -e {toxinidir}/instrumentation/opentelemetry-instrumentation-botocore[test]
   python -m pip install -e {toxinidir}/instrumentation/opentelemetry-instrumentation-boto3sqs[test]
-  python -m pip install -e {toxinidir}/instrumentation/opentelemetry-instrumentation-django[test]
+  pip install -r {toxinidir}/instrumentation/opentelemetry-instrumentation-django/test-requirements-3.txt
```

Comment:
Keep `python -m pip` as other packages?

Reply:
No, we are installing from a requirements file here and we don't need to call Python but pip directly